src/main/java/com/gtnewhorizons/angelica/compat/nd/Quad.java

```diff
@@ -31,10 +32,14 @@ public class Quad implements ModelQuadView {
 
     private final Vector3f vectorA = new Vector3f(), vectorB = new Vector3f(), vectorC = new Vector3f();
 
-    private boolean hasColor;
+    public boolean hasColor;
```

Comment:
why change the visibility? There's a getter, and a setter could be added if needed